frontend/src/components/molecules/ExpandableQuote/index.tsx

```diff
@@ -1,22 +1,37 @@
-import React, { useEffect, useState } from 'react';
+import React, { useEffect, useRef, useState } from 'react';
 import './style.css';
 import AlertIcon from '../../atoms/icons/alert';
 import { ArrowDownIcon, ArrowUpIcon } from '../../atoms/icons/fr-fi-icons';
 
-type Props = { title: string; large?: boolean; openOnMount?: boolean };
+type Props = {
+  title: string;
+  large?: boolean;
+  openOnMount?: boolean;
+  scrollOnMount?: boolean;
+};
 
 export const ExpandableQuote: React.FC<Props> = ({
   title,
   large = false,
   openOnMount = false,
+  scrollOnMount = false,
   children,
 }) => {
   const [expanded, setExpanded] = useState(false);
+  const ref = useRef(null);
+
+  useEffect(() => {
+    if (scrollOnMount) {
+      // @ts-ignore
+      ref.current.scrollIntoView();
+    }
+  }, [ref, scrollOnMount]);
 
   useEffect(() => setExpanded(openOnMount), [openOnMount]);
```

Comment:
Not in this PR scope but this useEffect is not required. The goal is to have `expanded` initially set to the `openOnMount` prop. `useState(openOnMount)` is sufficient line 20